satellite-software/platformio.ini

```diff
@@ -25,8 +25,8 @@ build_flags =
     -D LOG_LEVEL=3
     -D LMIC_DEBUG_LEVEL=0
 ;	-D ENABLE_LOGGER_SENSOR
-monitor_port = /dev/cu.usbserial-14310
-upload_port = /dev/cu.usbserial-14310
+;monitor_port = COM6
+;upload_port = COM6
```

Comment:
As said above, this is really your local build config and should not be part of a PR.